File: ResourceManager/Configuration/FilterColumns.vb
Imports System.Collections.Generic

Namespace ResourceManager.Configuration
    Public Module FilterColumns
        Public User As IEnumerable(Of String) = {"Name", "AccountName"}
        Public Tag As IEnumerable(Of String) = {"Name", "PasswordCount", "BookCount", "ArticleCount", "ActivityCount", "RepositoryCount", "NoteCount"}
        Public Role As IEnumerable(Of String) = {"Name", "Description"}
        Public Permission As IEnumerable(Of String) = {"Name", "Description"}
        Public Book As IEnumerable(Of String) = {"Title", "Author", "Publisher", "Abstraction", "TagString"}
        Public Article As IEnumerable(Of String) = {"Title", "Author", "TagString"}
        Public Password As IEnumerable(Of String) = {"Name", "UserName", "Url", "Description", "TagString"}
        Public Repository As IEnumerable(Of String) = {"Title", "FileType", "FilePath", "FileSize", "TagString"}
        Public Activity As IEnumerable(Of String) = {"Title", "Date", "StartTime", "EndTime", "Metric", "Quantity", "Output", "Note", "TagString"}
        Public Note As IEnumerable(Of String) = {"Title", "Date", "Notes", "TagString"}
    End Module
End Namespace

File: ResourceManager/Configuration/FilterColumns.cs
using System.Collections.Generic;

namespace ResourceManager.Configuration
{
    public static class FilterColumns
    {
        public static IEnumerable<string> User = new[] { "Name", "AccountName" };
        public static IEnumerable<string> Tag = new[] { "Name", "PasswordCount", "BookCount","ArticleCount","ActivityCount","RepositoryCount","NoteCount" };
        public static IEnumerable<string> Role = new[] { "Name", "Description" };
        public static IEnumerable<string> Permission = new[] { "Name", "Description" };
        public static IEnumerable<string> Book = new[] { "Title", "Author", "Publisher", "Abstraction", "TagString" };
        public static IEnumerable<string> Article = new[] { "Title", "Author", "TagString" };
        public static IEnumerable<string> Password = new[] { "Name", "UserName", "Url", "Description", "TagString" };
        public static IEnumerable<string> Repository = new[] { "Title", "FileType", "FilePath", "FileSize", "TagString" };
        public static IEnumerable<string> Activity = new[] { "Title", "Date", "StartTime", "EndTime", "Metric", "Quantity", "Output", "Note", "TagString" };
        public static IEnumerable<string> Note = new[] { "Title", "Date", "Notes", "TagString" };
    }
}
